### ISSUE TITLE
Harden transforms for model export

### ISSUE THREAD

**Issue by mauigna06** [PerkLab/SlicerOpenAnatomy#14]
Transforms applied to modelNodes should not be ignored while exporting

**Comment by lassoan** [comment]
Good point, we should apply the parent transform to the polydata of models/segments/planes before exporting. We don't even need to make it optional, but export should always be in world coordinate system.
